openedx/core/djangoapps/catalog/models.py

```diff
@@ -25,6 +27,14 @@ class CatalogIntegration(ConfigurationModel):
         )
     )
 
+    course_run_cache_ttl = models.PositiveIntegerField(
```

Comment:
Why is it necessary to introduce a separate TTL for course run data? I would prefer to keep this config model as simple as possible, and I don't see the benefit to introducing separate TTLs for different types of data coming from the catalog. If the existing TTL isn't high enough, let's talk about why that is. FYI @clintonb @mikedikan